### ISSUE TITLE
🧑‍🦼 Events template - Needs a h1

### ISSUE THREAD

**Issue by bradystroud** [SSWConsulting/SSW.Website#1398]
cc @jaydenalchin @hveraus @sethdailyssw @tiagov8 

Every page needs a h1 as per https://www.ssw.com.au/rules/make-title-h1-and-h2-tags-descriptive/

This is important for accessibility and SEO. See WCAG 2.4.6
https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-descriptive.html

WCAG 1.4.5 is also relevant
https://www.w3.org/WAI/WCAG21/Understanding/images-of-text.html

```[tasklist]
### Tasks
- [x] Add a h1 on top of the banner. This should be centered and ~black or white~ sswRed. H1 styling should be consistent with SSW Rules
- [x] Checked by @jaydenalchin
- [x] Check any other pages for the same issue
```

**Comment by amankumarrr** [comment]
@wicksipedia @bradystroud @sethdailyssw @jaydenalchin @hveraus 

As per email with the subject "Re: SSW Event pages - Banner redesign ", we are going to implement the following design for all events pages.



![Image](https://github.com/SSWConsulting/SSW.Website/assets/71385247/c2cc11c8-d4b8-4ae8-a05f-cd5099eab55e)

**Figure: Events' image and h1 under the banner with text**

**Comment by sethdaily** [comment]
@amankumarrr 
As per my conversation with @camillars and @ae-cher-ssw
It's better to keep the banners as they are now, instead of getting photos (even though it kind of violates DRY)

Done - I have removed the task 'change the banners to images from Google Photos'

**Comment by amankumarrr** [comment]
@sethdailyssw @camillars @ae-cher-ssw @wicksipedia @bradystroud @adamcogan @hveraus @jaydenalchin 

Hi Team,

With that change, our events page will look like this:
<img width="1265" alt="image" src="https://github.com/SSWConsulting/SSW.Website/assets/71385247/dc14e886-b776-490c-8e93-0e92e7180183">

**Figure: Angular Workshop page with new h1**

**Comment by camillars** [comment]
@amankumarrr 

1. please align the content to the left

thanks

**Comment by amankumarrr** [comment]
Hi @camillars,

Our website has all the content centered aligned, it would be inconsistent with that alignment, do you still want to keep it left aligned?


<img width="1258" alt="image" src="https://github.com/SSWConsulting/SSW.Website/assets/71385247/a17be9a7-6cad-4829-8b8b-8a2e3cee465a">

**Figure: With left indented text**

**Comment by camillars** [comment]
@amankumarrr 

Only the consulting pages AFAIK - everything else is aligned to the left.

can the events follow the same rule/standard?

**Comment by sethdaily** [comment]
@amankumarrr @camillars
+1 for making all events pages left aligned

**Comment by amankumarrr** [comment]
@sethdailyssw @camillars @ae-cher-ssw @bradystroud @wicksipedia 

Changes are live now.

<img width="920" alt="MicrosoftTeams-image (9)" src="https://github.com/SSWConsulting/SSW.Website/assets/71385247/464d8bb9-edef-4afe-b38d-11fc86c3652d">


**Figure: Angular Workshop page with new h1 and left aligned text**


I found the following pages which do not have h1. 

- Events
   - [Scrum](https://www.ssw.com.au/events/professional-tailored-scrum-training)
   - [Customized Courses](https://www.ssw.com.au/events/customized-courses)
- All offices 
- Homepage 
- Newsletters 

and here is the issue for this #1612, closing this issue.
